src/main/webapp/mentors.js

```diff
@@ -237,8 +237,8 @@ function getProjectName(projectID) {
       'Encountered a server error, please try again later.')
       .then((response) => response.json()).then((data) => {
         console.log('got project data');
-        console.log(data.previewData.name);
-        return data.previewData.name;
+        console.log(data[0].name);
```

Comment:
Thanks for going ahead and changing this so it doesn't break!

Reply:
Oh cool, dashboard.js also uses previewData, could you maybe make the change there too?

Reply:
@Obinnabii Do [these changes](https://github.com/googleinterns/step22-2020/pull/126/files#diff-d5bd38389c6679a39a26096d66d07de0) cover it?